multi-arch-builders/provisioning/aarch64/README.md

```diff
@@ -0,0 +1,104 @@
+# OpenTofu
+
+ OpenTofu is a Terraform fork, is an open-source infrastructure as code (IaC) tool
+ lets you define both cloud and on-prem resources in human-readable configuration files
+ that you can version, reuse, and share.
+
+ To proceed with the next steps, ensure that 'tofu' is installed on your system.
+ See: https://github.com/opentofu/opentofu/releases
+
+## Before starting
+
+### AWS credentials
+
+```bash
+# Add your credentials to the environment.
+# Be aware for aarch64 the region is us-east-2
+HISTCONTROL='ignoreboth'
+ export AWS_DEFAULT_REGION=us-east-2
+ export AWS_ACCESS_KEY_ID=XXXX
+ export AWS_SECRET_ACCESS_KEY=YYYYYYYY
+```
+
+Make sure your AMI user has access to this policies:
+
+```json
+{
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Sid": "VisualEditor0",
+			"Effect": "Allow",
+			"Action": [
+				"ec2:TerminateInstances",
+				"ec2:StartInstances",
+				"ec2:CreateTags",
+				"ec2:StopInstances"
+			],
+			"Resource": "arn:aws:ec2:*:*:instance/*"
+		},
+		{
+			"Sid": "VisualEditor1",
+			"Effect": "Allow",
+			"Action": [
+				"ec2:DescribeInstances",
+				"sts:DecodeAuthorizationMessage"
+			],
+			"Resource": "*"
+		},
+		{
+			"Sid": "VisualEditor2",
+			"Effect": "Allow",
+			"Action": [
+				"ec2:RunInstances",
+				"ec2:CreateTags",
+				"ec2:CreateVpcEndpoint",
+				"ec2:CreateKeyPair",
+				"ec2:CreateSecurityGroup",
+				"ec2:CreateSubnet"
+			],
+			"Resource": [
+				"arn:aws:ec2:*:*:instance/*",
+				"arn:aws:ec2:*:*:key-pair/*",
+				"arn:aws:ec2:*:*:security-group/*",
+				"arn:aws:ec2:*:*:volume/*",
+				"arn:aws:ec2:*:*:image/ami-*",
+				"arn:aws:ec2:*:*:network-interface/*",
+				"arn:aws:ec2:*:*:subnet/*"
+			]
+		}
+	]
+}
```

Comment:
To keep things simple I would just say your user/credential needs access to ec2 and have a policy that looked like:

```
{
	"Version": "2012-10-17",
	"Statement": [
		{
			"Sid": "VisualEditor0",
			"Effect": "Allow",
			"Action": [
				"ec2:*"
			],
			"Resource": "*""
		}
	]
}
```